Tasks/count.js

```diff
@@ -1,17 +1,12 @@
-// Sum all number values in dict
+'use strict'
 
-count = (obj) => {
-  sum = 0;
-  {
+ const adder = (obj) => {
+  let sum = 0;
+  const arr_of_values = Object.values(obj);
```

Comment:
```suggestion
  const arrValues = Object.values(obj);
```
You should change the name of variable, the snake case is't good